crates/ruff/src/registry.rs

```diff
@@ -347,6 +347,13 @@ impl Rule {
             _ => LintSource::Ast,
         }
     }
+
+    /// Return the URL for the rule documentation, if it exists.
+    pub fn url(&self) -> Option<String> {
+        self.explanation()
+            .is_some()
+            .then(|| format!("https://beta.ruff.rs/docs/rules/{}", self.as_ref()))
```

Comment:
Does it make sense to use `env!("CARGO_PKG_HOMEPAGE")` here to have a single place to change the url? (could also be it's less confusing to have rust not depend on toml metadata)